[2025-07-12T14:21:25.987+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flight_data_etl.load_to_database manual__2025-07-12T14:21:21.439463+00:00 [queued]>
[2025-07-12T14:21:25.990+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flight_data_etl.load_to_database manual__2025-07-12T14:21:21.439463+00:00 [queued]>
[2025-07-12T14:21:25.990+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-07-12T14:21:25.995+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_to_database> on 2025-07-12 14:21:21.439463+00:00
[2025-07-12T14:21:25.998+0000] {standard_task_runner.py:60} INFO - Started process 355 to run task
[2025-07-12T14:21:25.999+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'flight_data_etl', 'load_to_database', 'manual__2025-07-12T14:21:21.439463+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/flight_data_etl.py', '--cfg-path', '/tmp/tmpex6e4bo8']
[2025-07-12T14:21:26.000+0000] {standard_task_runner.py:88} INFO - Job 57: Subtask load_to_database
[2025-07-12T14:21:26.019+0000] {task_command.py:423} INFO - Running <TaskInstance: flight_data_etl.load_to_database manual__2025-07-12T14:21:21.439463+00:00 [running]> on host b78d8dd28f9c
[2025-07-12T14:21:26.127+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineer' AIRFLOW_CTX_DAG_ID='flight_data_etl' AIRFLOW_CTX_TASK_ID='load_to_database' AIRFLOW_CTX_EXECUTION_DATE='2025-07-12T14:21:21.439463+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-12T14:21:21.439463+00:00'
[2025-07-12T14:21:26.132+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-07-12T14:21:26.146+0000] {logging_mixin.py:188} INFO - Loaded 10 airport records
[2025-07-12T14:21:26.151+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-07-12T14:21:26.178+0000] {logging_mixin.py:188} INFO - Loaded 1000 scheduled flight records
[2025-07-12T14:21:26.195+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-07-12T14:21:26.499+0000] {logging_mixin.py:188} INFO - Loaded 11353 real-time flight records
[2025-07-12T14:21:26.499+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-07-12T14:21:26.503+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=flight_data_etl, task_id=load_to_database, execution_date=20250712T142121, start_date=20250712T142125, end_date=20250712T142126
[2025-07-12T14:21:26.525+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-12T14:21:26.532+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
